engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -18,16 +18,21 @@
  *   within the table.
  */
 angular.module('Bastion.errata').controller('ErrataController',
-    ['$scope', '$state', '$location', 'translate', 'Nutupane', 'Erratum', 'IncrementalUpdate', 'Repository', 'CurrentOrganization',
-    function ($scope, $state, $location, translate, Nutupane, Erratum, IncrementalUpdate, Repository, CurrentOrganization) {
+    ['$scope', '$state', '$stateParams', '$location', 'translate', 'Nutupane', 'Erratum', 'IncrementalUpdate', 'Repository', 'CurrentOrganization',
+    function ($scope, $state, $stateParams, $location, translate, Nutupane, Erratum, IncrementalUpdate, Repository, CurrentOrganization) {
         var nutupane, params = {
             'organization_id': CurrentOrganization,
             'search': $location.search().search || "",
             'sort_by': 'updated',
             'sort_order': 'DESC',
             'paged': true,
-            'errata_restrict_applicable': true
+            'errata_restrict_applicable': false,
+            'disableAutoLoad': true
```

Comment:
Why did you need to set `disableAutoLoad`?

Reply:
I was seeing 2 API calls being made to errata controller on opening the page. This flag tells nutupane not to load the records when new Nutupane is called. Instead wait till all params are filled in the controller and load when refresh is called towards the end.